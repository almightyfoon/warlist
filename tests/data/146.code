c4501b_-0B0E191h1h1hi22ki5i5hZh_h-h_bB__0Z19191h1h0t2k2k9u1N1P1N1P26