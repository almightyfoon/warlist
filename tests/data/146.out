https://conflictchamber.com/#c4501b_-0B0E191h1h1hi22ki5i5hZh_h-h_bB__0Z19191h1h0t2k2k9u1Nke1Nke

Cryx Army - 75 / 75 points
[Theme] Dark Host

[Asphyxious 2] Lich Lord Asphyxious [+28]
 - Deathripper [6]
 - Stalker [8]
 - Stalker [8]
 - Stalker [8]
Bane Lord Tartarus [0(6)]
Machine Wraith [2]
Bane Knights (max) [15]
Bane Knights (max) [15]
Bane Warriors (min) [10]
 - Bane Warrior Officer & Standard [0(5)]
Bane Warriors (max) [16]
 - Bane Warrior Officer & Standard [0(5)]
Wraith Engine [15]

Cryx Army - 59 / 75 points

!!! This army must contain at least 96 total points.

[Coven 1] The Witch Coven of Garlghast [+26]
 - Deathripper [6]
 - Deathripper [6]
 - Stalker [8]
 - Stalker [8]
 - The Withershadow Combine [9]
Machine Wraith [2]
Machine Wraith [2]
Ragman [4]
Satyxis Raiders (max) [16]
 - Satyxis Raider Sea Witch [4]
Satyxis Raiders (max) [16]
 - Satyxis Raider Sea Witch [4]
